examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -0,0 +1,1360 @@
ys69mH4mWROJMcfhCPuq8deGq3SE5hFqtVlbXSTIWvYV4czwGrQG7x47L78IT9FDjriFKH4Uv5Dshk8x7XPQXAILxsdRdfsGvfj5B+C2IniFBOB3ZbMoE7OMVHa30OA0/ONH5vfdg6VK47bbIuBEjlH8nTlSG6TIzlaRozZrIHpgVK5Q5Sx98AFdfjWrZMrJiszBoDbj8LsrGX83GD16k8pzG3ahd7x5DcqRt4e+0+fPhrrtavm78eCWJMxxWR8nhUMoZfPNNq1/SXFLJqJE3hh9H5xYx6JHGZDR29wFG/nUaAwZezOjzJ5L879mo610QCkU8j/6c0ZgbfMTXBxkw87Nmv1bqY88TU1aHubw2fKzfwAvpPupq1E4XGocTQ+4BCIWoc1SQYmxHgjkBzYcfEffWXCpcFXT4bAnd3v6cWncNNa4q2tV4Ofulz0lY/BN7y3dQ7CwmJIXoGt+VKH0UBo2BpLxK+maPpesXP9ExtiPDK/QMf+o/pNYr84p8ddVcfsMTjL5sMvEffo5t2c+tttlRBRt7qiybdjQ9L8ut9ogpITK1nlocXke4UKcgnGhiztBRiDlDQptUX6+szLr00qaJicsFQ4YoPVM//xxZUwnlzeunwp8oc5aRak1FRibk9zFqznLU545GHj4c960T8AzsQ8K8hXD4xG2A664DiwXefVd5fNddyhvmkCFw003KsSMTQb1eKVGwYYPSI7R9O1xxRcur6o6H0djiPCtXdl8kgx7b/yJ7sbyZaeiqatE0uJu9rjVld/2V+M++Rl9epTy/vQ6zX6Z2+XckduwDwO5XHqP7vcrqwr3T7iLl/flYC8rCz+GPseJPjCdoNKD/ZD4lCXqqPdX0v24ypv+tVYLWrVOGTYuKCCYnsfC7l+n01Sr6PRbZG9nQvyfFj91HxYCuuNasJKHvcKKKypGMRnydMkibNgPJbKR0yh1Nvi+G/CJ6n3klAOU3XUXJMw83nttfQKdb/45kNrH7y/dAo8Fnr6FO9pAU056QHEKr1mL32sN70Tl9TmRkEi2Jx91rVe4qx6wzN18sVTipToU5QyIZOgqRDAlCM2S51Z6pLeVb2FaxjR7teigruKQQZ3Q4o2ngzJlw993K56NHKwlQWhpUVioTyDt1gjffbLrNyKJF8Oyz8NNPStzDDyuTzC+6qDGmokIZ5mvOvHnw/fewaZOSQPXtC1u3Nh973XXK0ODmzQB4xt+A7uNP0Xr9zce3wvHs4zD7PaL3FBw9+EhDhsDatcd9mZydjW9QP+zjryVp4v2otm1vNs5vi0KlAp2j6Yq6w9kH9CR62x5UwRD280YSs/QnAHb+exqeqy5FU2snZI3CUFhC+2dnEvvtMgDKz+hH7oczlaX8skz3SyeEe4uCZiMlj0wm7fEZ1IwYwMqnJzJk2lv4xpzD9kuG0DuxN7Er1uL88D32P3InNfoAQ7/ciK+8BPvf70Kr0RGSQrgDbqyGxuHjhLmfoS+toOCBW6n0VOH0Oeme0P242zD1udcxb9lJ3qwXmgxjngzugBtJlojStz6373QhkqHTgEiGBOH4FdgLWF28mv7J/XH7lV/c2e2zmwZKErz6qvJGf2ho7nhIkpKYaVqoETR4cGM9psMFg43XBAJKXSWPRyl2CfDPfzYOB86dC9u2KSUKgIbP/suqDip6zphD+jxlA96gyYDWc9hy+n79kK68gqqhfUh4fiaaZcuVuAN55MjFRK/fRq+n3sGbGItcUY5ly67wpZ6keGSzCVe/HiR+sbTl165SNR1iuvFGeP99pWbVgw9GnhszRllxV1tLa+QuXSibeg+pE+5uNa7Za1UqVLKMp0sW+gNFaAKNw2Te9BR+/H4WXX7YRJc7Hmn1eep7dMK2S+nV2/robXTfWIj+G2VVYfHE69k76U+cO0Dpcdp61zhScivIu/lKEj7+CvetE4hduZZAuwSy7psOwOq5TxM46wyq3dXIskw8ZqwxkQsNtNWxBX5pAAAcg0lEQVS1WDZuQ1dZg7dTBq5hA0l84l+kfPwtWkc9AAdmPEYgMYH6UcMw7dyLt1MmsqnlHqroRcvJmPI0+a89hfPMwcfVlrryKtrN/oSqm64m0D4yqS9wFKB1NpBuTkGKiz2u5z0ViWToNCCSIUE4fjXuGnKKc+iW0I2smCyAYypqeMI5HMq2JI89plTf3rlTmf90xRXNx7/2mjIv6t13lc9zcmD2bKUH6gJl8q+nMI+V/r3ES0Z6XTUR45481i6YSfe0/kTvK1SGntSR0zFr35iBRW/B8H8T8QQ86DQ6tGpl+NH30osY7le2Xznw+H1U/d/15Nvz0ajUdC10k75+HzGPPtn0Xg/rsZLj4ghWloe3JAGUYbAhQ5ped8hll0FJCfztb1BXR82+rdQO7k2Xa26jXvbi3rWN5AFnHnNTH42sVrPtgb/Q54W5qH7F2463f28qpz1AhysnHPM15ReeiXvuuxTU5tPvgReIXryCynFjCXTthKG4HFQq2s2dj/pglfdgtJWdH75C30tubvb5XAP7ELVxGwCBuBgqb7me8jvHIzudhFauwDB0BMmz/kvyG/8JX7Nz6X/x9OwSfqyx1xOKUd5TGvwNqFXqiAKYXf50O7b/raOhf092fzMXTZ2DUGw0noAHp6+eUWNuwVBRTdETD1Dzl6sbb06Wif/vlwTbxSOrVdSfe0ZEL64hr5DopSup/svVp0QvF4hk6LQgkiFBOH7+kJ81xWuIN8fTs13Pk307v14goCRQiYmE3p1FTnEOsaZYekZl4S4tZKdR2SxWr9Ef/3P7/bhvuh7j0mXIOTl4M9qzrnQdRo2RgakD0dc3wHnnKXvRdeqkbMtyaEVddbVSIuHBB5WNfA8XCjU/Ef2SS+A//2ky9CjLMjIyalVjIhf6fim5//sK28ocCh+7l94v/gfL14sjrguajTT06U70ms0RxyWdlkXLZpFS6SF9xiwSVm1ofMnxMTgTY0CjJn57XsR1gT690G1rZrL1ofv
```

Comment:
I'm confused by what " in the paragraph above" refers to. Maybe remove it, or rephrase the reference.

Reply:
"You can replace the device definition with the commented line to check this". I suggest removing this sentence, as it gives the impression that there is one more thing needed to be done, but really there isn't. It's enough the change the device ARN, which you already explained in the previous sentence.

Reply:
Code examples would be nice, even if you only run it for 1 iteration.

Reply:
@licedric The 29-qubit test is with the same script of qaoa_stress_test.py. Added a sentence to explicitly state it.